cmd/database/database_create.go

```diff
@@ -133,15 +133,26 @@ var dbCreateCmd = &cobra.Command{
 
 		if !softwareVersionIsValid {
 			if softwareVersion == "" {
-				utility.Error(fmt.Sprintf("No version specified for %s. Please provide a version using --version flag. For example, civo database create db-psql --software psql --version 14", canonicalSoftwareName))
+				utility.Error("No version specified for %s. Please provide a version using --version flag. For example, civo database create db-psql --software psql --version 14", canonicalSoftwareName)
 			} else {
-				utility.Error("The provided software version is not valid. Please check the available versions for the specified software.")
+				utility.Error("The provided software version is not valid. Please check the available versions for the specified software")
```

Comment:
Always tell the user how to do what you are suggesting.

"The provided software version is not valid. Please check the available versions by typing civo db engine"